microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,85 @@
+
+## Policy Controls 
+A related new policy added to Microsoft Edge provides a setting for corporate or enterprise-level default configurations: **ShareOSRegionThroughJavaScriptLocale** which accepts an integer with potential values: 
+
+|Number| Description |
+|--------|-------------------------|
+| 1 | Limited | 
+| 2 | Always | 
+| 3 | Never | 
+
+In addition, an [older policy](/DeployEdge/microsoft-edge-policies#applicationlocalevalue) (**ApplicationLocaleValue**) allows setting the Microsoft Edge locale and prevents users from changing it. 
+
+## Avoiding Potential Misuse 
+We believe that **limited** is the right default that allows the most customer’s regional preferences to be respected with the least amount of risk of the website misusing this information.
+
+#### Fingerprinting Entropy
+OS regional information could be misused to compromise the user's privacy.  The regional information would help hackers establish fingerprinting entropy. In combination with many other pre-existing bits of entropy in the platform today, the user could be uniquely identified leading to more potential secondary harm. While a user's region and language are already available via the web platform, users generally fall into well-recognized regional codes that are similar across geographies.  Because the sample sizes are so large, the risk of any specific regional code standing out is small, and thus the risk is small. If the user changes their OS preferences to a unique language + region, websites may be able to identify the anomaly in a population, and easily uniquely identify the associated customer. 
+
+#### Content Presentation
+The OS regional information could be misused by a website is in the consistency of their content presentation. There are various places in the web platform where language and regional information is made available. Regional information is processed in one place, while language preferences are surfaced in another.  Furthermore, this information is proactively sent to the server on the initial request for content. When the language and regional information is self-consistent, then it doesn’t really matter which signals the website uses to assemble its content presentation. If one part of the web platform starts reporting a language preference, but another part of the platform reports a separate language, then the website may start to mix-and-match regional preferences resulting in strange and unusual experiences.  Watch for the phrasing of requests that mixes languages and regional preferences together (such as "Email sent on miércoles, 5 de enero de 2021").
```

Comment:
Global in page: try 2 or 3 sentences per paragraph max; add whitespace blank lines to group sentences even if arbitrary.